css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
```

Comment:
Hmmm, this comment makes it seem like there should be code here from you, but none was added. Could there be a mention of something you should do here in the instructions? 🤔